src/lib/eventListeners.js

```diff
@@ -16,9 +16,10 @@
  */
 
 const eventsMap = new Map()
+const cache = new Map()
```

Comment:
can we rename this to `callbackCache`? If I understood the code correctly, this map caches the callbacks., right?

Reply:
yes, the map caches the callbacks and I agree it would be better to have non-generic variable name there. I will change it.

Reply:
renamed the variable.